test/verify/check-storage-basic

```diff
@@ -69,8 +69,12 @@ class TestStorageBasic(storagelib.StorageCase):
         b.wait_visible(self.card("Storage"))
         b.wait_visible(self.card_row("Storage", name=disk))
         b.assert_pixels(self.card("Storage"), "overview",
-                        # Usage numbers for the root filesystem are not stable
-                        ignore=["[data-test-row-location='/']"])
+                        # Usage numbers are not stable and also cause
+                        # the table columns to shift. The usage bars
+                        # are not stable but are always the same size,
+                        # so it is good enough to ignore them.
+                        mock={".usage-text": "---"},
```

Comment:
I'd have prefered a realistic mock value which represents the right size, like "1.2 / 34 GB" or so. That should better point out wrapping problems with narrow layouts.

But details, good enough!